wiki/Glossary/en.md

```diff
@@ -560,18 +550,19 @@ Activities are shown either at [Works In Progress/Help](https://osu.ppy.sh/forum
 
 NC has two meanings:
 
-1. Short for "New Combo", a modding term to suggest that a beat needs to be the start of a new combo.
-2. An abbreviation for "Nightcore" mod, a variant of the double time mod that increases the pitch of the music and adds a consistent bass drum beat to the music.
+1. short for new combo, a modding term to suggest that a beat needs to be the start of a new combo
+2. an abbreviation for nightcore mod, a variant of the double time mod that increases the pitch of the music and adds a consistent bass drum beat to the music.
+
+### Nuke
 
-### NM
+A beatmap that is really far from completion is considered to be nuked.
```

Comment:
"Any beatmap that is considered to be disregarding the rules (or the mapper disregarding the Code of Conduct) can be nuked. Nuked beatmaps are considered "closed" until significant changes are made and the status is manually removed by a member of the team."